src/_main_/utils/pagination/__init__.py

```diff
@@ -0,0 +1,24 @@
+from django.core.paginator import Paginator, EmptyPage
+
+from _main_.utils.common import serialize_all
+
+def paginate(queryset, page, limit):
```

Comment:
should we have default limit=50 if not specified?

Reply:
if limit not passed in args, this would be passed as null.  Let's protect against that with default=50